tests/unit/index.ts

```diff
@@ -82,12 +82,37 @@ registerSuite({
 			assert.include(contents, `from 'sub/baz';`);
 		});
 	},
+	'test prefixing of absolute paths with options.name': function () {
+		// we reuse foo-directories from the directory stress test above.
```

Comment:
See other comment below about the change to main... it's probably cleaner as a separate test file that still leverages test/support/foo-directories, while running it through a clean set of tests, though that could result in a fair amount of duplication so I'm not completely against this approach.